gradebookng/tool/src/java/org/sakaiproject/gradebookng/tool/panels/importExport/GradeImportUploadStep.java

```diff
@@ -99,20 +99,20 @@ public void onSubmit() {
 				try {
 					spreadsheetWrapper = ImportGradesHelper.parseImportedGradeFile(upload.getInputStream(), upload.getContentType(), upload.getClientFileName(), userMap);
 				} catch (final GbImportExportInvalidColumnException e) {
-					log.debug("GBNG import error", e);
-					error(getString("importExport.error.incorrectformat"));
+					log.debug("incorrect format", e);
+					error(getString("importExport.error.incorrectformat")+" - "+e.getMessage());
 					return;
 				} catch (final GbImportExportInvalidFileTypeException | InvalidFormatException e) {
-					log.debug("GBNG import error", e);
-					error(getString("importExport.error.incorrecttype"));
+					log.debug("incorrect type", e);
+					error(getString("importExport.error.incorrecttype")+" - "+e.getMessage());
 					return;
 				} catch (final GbImportExportDuplicateColumnException e) {
-					log.debug("GBNG import error", e);
-					error(getString("importExport.error.duplicatecolumn"));
+					log.debug("duplicate column", e);
+					error(getString("importExport.error.duplicatecolumn")+" - "+e.getMessage());
```

Comment:
Thanks Steve for looking.
getString("importExport.error.duplicatecolumn") is localized would it not have made more sense to just remove the e.getMessage in a new PR instead of reverting?